mobile/library/cc/stream_prototype.cc

```diff
@@ -10,10 +10,11 @@ StreamPrototype::StreamPrototype(EngineSharedPtr engine) : engine_(engine) {
 }
 
 StreamSharedPtr StreamPrototype::start(bool explicit_flow_control) {
-  auto envoy_stream = init_stream(engine_->engine_);
-  start_stream(engine_->engine_, envoy_stream, callbacks_->asEnvoyHttpCallbacks(),
+  envoy_engine_t engine_handle = reinterpret_cast<envoy_engine_t>(engine_->engine_);
+  auto envoy_stream = init_stream(engine_handle);
```

Comment:
it looks like `init_stream` isn't using the `engine_handle` param, should we just remove it from the function?

Reply:
arguably but I'd rather just remove the function in my follow-up =P

Reply:
SGTM!